teachertool/src/utils/storageHelpers.ts

```diff
@@ -0,0 +1,21 @@
+import { Rubric } from "../types/rubric";
+import * as LocalStorage from "../services/localStorage";
+import * as IndexedDb from "../services/indexedDbService";
+
+export async function getLastActiveRubricAsync(): Promise<Rubric | undefined> {
+    const lastActiveRubricName = LocalStorage.getLastActiveRubricName();
+    return await IndexedDb.getRubricFromIndexedDbAsync(lastActiveRubricName);
+}
+
+export async function saveRubricAsync(rubric: Rubric) {
```

Comment:
I think these should just be transforms.

Reply:
I can move it. I wasn't sure since they don't really touch app state and I don't think transforms usually return things...but I'm happy to move them if you think that's a better fit. I'm pretty ambivalent.

Reply:
I guess these are a bit different than transforms. I want to avoid the unfettered propagation of helper methods around the codebase. We have them in state, and now in utils. It will become messier over time. Perhaps move them all to a `helpers` folder? I like that solution. Must contain the chaos. Fine as is for now. We can move them in a separate pr.

Reply:
I was under the impression that's sort of what `utils` was :)

Reply:
Do these helpers represent a storage service? Maybe that's the right place for them.

Reply:
I agree; I view things in utils folders as things that are supposed to help in some way. I think the `utils` folder is the right place for this.

Reply:
In my mind, the definitions go like this:
* **util**: is a very basic utility function. It shouldn't have to call into services. Example: `debounce`.
* **helper**: makes reading or writing data easier, handling common edge cases internally. It can access services. Example: `getSafeRubricName`.
* **transform**: modifies application state.

Reply:
I'm fine with any of these solutions.

Reply:
I'm partial to 3!